src/styles/dialog.css

```diff
@@ -3,8 +3,8 @@
  * @author Jamie jamie.h@inwinstack.com
  */
 
-.md-confirm-dialog-warn-theme md-dialog-actions button:last-child {
-  color: #FF6D00;
+.md-default-theme md-dialog-actions button:last-child {
```

Comment:
I think we use an incorrect method to the theme. To avoid getting stuck of the project progress, please revert this change and remove the `.theme()` until we find out the correct usage of the theme.